proof/refine/AARCH64/ArchKHeap_R.thy

```diff
@@ -0,0 +1,413 @@
+(*
+ * Copyright 2023, Proofcraft Pty Ltd
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+theory ArchKHeap_R
+imports
+  KHeap_R
+begin
+
+context Arch begin arch_global_naming
+
+named_theorems KHeap_R_assms
+
+lemma getObject_inv_vcpu[wp]:
+  "\<lbrace>P\<rbrace> getObject l \<lbrace>\<lambda>(rv :: ArchStructures_H.vcpu). P\<rbrace>"
```

Comment:
Very nitpick but I think the more standard style for this would be
```suggestion
  "\<lbrace>P\<rbrace> getObject l \<lbrace>\<lambda>_::ArchStructures_H.vcpu. P\<rbrace>"
```

Reply:
done, will show up in next push

Reply:
You might have already done it, but would want to do that for all architectures

Reply:
done